Sources/spm-test-finder/main.swift

```diff
@@ -0,0 +1,48 @@
+#if os(OSX)
+
+import XCTest
+import Foundation
+
+enum Error: ErrorProtocol {
+    case invalidUsage
+    case unableToLoadBundle(String)
+}
+
+do {
+    guard Process.arguments.count == 2 else {
+        throw Error.invalidUsage
+    }
+    var bundlePath = Process.arguments[1]
+    if !(bundlePath as NSString).isAbsolutePath {
+        bundlePath = NSFileManager.default().currentDirectoryPath + "/" + bundlePath
+    }
+    bundlePath = (bundlePath as NSString).standardizingPath
+
+    guard let bundle = NSBundle(path: bundlePath) where bundle.load() else {
```

Comment:
I would prefer that we just make the tool take an argument about where to write its output.